<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistDialog</class>
 <widget class="QDialog" name="RegistDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>231</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>344</width>
    <height>231</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>344</width>
    <height>243</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>用户注册</string>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>WoCao进销存管理系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>53</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>53</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>账号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>53</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>53</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwdEdit1">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>53</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>53</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>重复密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwdEdit2">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="registButton">
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
